src/test/resources/application.properties

```diff
@@ -1,4 +1,9 @@
-spring.datasource.driver-class-name=org.h2.Driver
```

Comment:
Zwóć uwagę, że mamy 2 pliki application.properties.
Jeden w katalogu src/main w którym usunełaś:
spring.jpa.hibernate.naming.strategy
I to jest jak najbardziej ok.
Natomiast  drugi jest w katalogu src/test.
Ten w src/test nie powinien być zmieniony, mają tam być nadal namiary na baze H2, bo inaczej nie przjdą testy.

Reply:
I ten komentarz wiele wyjaśnił...